pipelinewise/cli/config.py

```diff
@@ -284,20 +290,22 @@ def save_tap_jsons(self, target, tap, extra_config_keys={}):
             if indices:
                 schema_mapping[source_schema]['indices'] = indices
 
-        # Schema mapping is ready
-        tap_schema_mapping = {
-            "schema_mapping": schema_mapping
-        }
-
         # Generate tap inheritable_config dict
         tap_inheritable_config = utils.delete_empty_keys({
-            "batch_size_rows": tap.get('batch_size_rows'),
-            "hard_delete": tap.get('hard_delete', True),
-            "flush_all_streams": tap.get('flush_all_streams', False),
-            "primary_key_required": tap.get('primary_key_required', True),
-            "default_target_schema": tap.get('default_target_schema'),
-            "default_target_schema_select_permissions": tap.get('default_target_schema_select_permissions'),
-            "schema_mapping": schema_mapping,
+            'batch_size_rows':
+                tap.get('batch_size_rows'),
```

Comment:
Hey Peter, what's up with the line breaks in this dictionary?

Reply:
the formatter is strict, and reformatted because it decided it's not pep8/google format compliant and the structure is deep enough for a line break.

IDEs usually accept multiple formatting, for example any of these are valid in pycharm:
```
a = {'abc': 123}
b = {
    'abc': 123
}
c = {
    'abc':
        123
}
```

Do we want to have this freedom or we want to enforce exactly the same coding style everywhere? Using yapf and other formatter maybe too strict :S what do you think?

Reply:
Or should we enforce only two things, 120 char max length and single-quotes in the code? And any other formatting task is the developer's preference.

Reply:
Hmm i think it's a preference, I find the unnecessary line breaks weird, especially this format:
```
c = {
    'abc':
        123
}
```

Reply:
@Samira-El fixed by fb81bd8b66cf43e026a90b273d228ff6beade99a and reverted every dict/list and function line-break formatting.

Also, a5c25d3ac40e40d6dbc4f59bd4221deef566bbe0 removing **yapf** formatter completely from circleci, so we don't force any strict formatting.

Circleci runs only unify (to remove double quotes) and pylint.

what do you think?

Reply:
Great!